specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/rbac.json

```diff
@@ -0,0 +1,610 @@
+            "schema": {
+              "$ref": "#/definitions/SqlRoleDefinitionCreateUpdateParameters"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Role Definition create or update request was completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/SqlRoleDefinitionGetResults"
+            }
+          },
+          "202": {
+            "description": "The Role Definition create or update request was accepted and will complete asynchronously."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/DefaultErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "operationId": "SqlResources_DeleteRoleDefinition",
+        "x-ms-examples": {
+          "CosmosDBSqlRoleDefinitionDelete": {
+            "$ref": "./examples/CosmosDBSqlRoleDefinitionDelete.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "description": "Deletes an existing Azure Cosmos DB Role Definition.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/roleDefinitionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The Role Definition delete operation was accepted and will complete asynchronously."
+          },
+          "204": {
+            "description": "The Role Definition delete operation was completed successfully."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/DefaultErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/sqlRoleDefinitions": {
+      "get": {
+        "operationId": "SqlResources_ListRoleDefinitions",
+        "x-ms-examples": {
+          "CosmosDBSqlRoleDefinitionList": {
+            "$ref": "./examples/CosmosDBSqlRoleDefinitionList.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "description": "Retrieves the list of all Azure Cosmos DB Role Definitions based on the filter.",
+        "parameters": [
```

Comment:
Can we not list without providing the filter?

Reply:
Since it is not implemented yet by RP, I have removed it